INSTALL.md

```diff
@@ -0,0 +1,233 @@
+with-embedded-ssl      Enable built-in SSL support                 true
+                       (required for DHX/RANDNUM authentication)
+with-gsaapi            Enable gssapi support                       true
+                       (required for Kerberos V authentication)
+with-kerberos          Enable Kerberos V API                       true
+                       (optional for Kerberos V authentication)
+with-krb5-uam          Compile Kerberos V UAM                      true
+with-largefile         Enable largefile support                    true
+with-ldap              Enable LDAP support                         true
+with-libiconv          Enable iconv support                
+with-manual            Compile Netatalk html documentation         false
+with-overwrite         Overwrite existing configuration files      false
+with-pam               Enable PAM support                          true
+with-pgp-uam           Compile PGP UAM                             false
+with-quota             Enable Quota support                        true
+with-sendfile          Enable sendfile support                     true
+with-shadow            Enable shadow password support              true
+with-overwrite         Overwrite existing configuration files      false
+with-spotlight         Enable Spotlight support                    true
+with-tcp-wrappers      Enable TCP wrappers                         true
+with-tests             Compile tests                               false
+with-zeroconf          Enable Zeroconf (Bonjour) support           true
+```
+
+If meson cannot find the some required dependencies at configure time then the
+user can specify the path a library if it is installed in an unconventional
+location.
+All such options take the form -`Dwith-dependency-path=/path/to/dependency`,
+and the dependency directory must include lib and include directories.
+
+The following dependencies can be found in this way:
+
+```
+gssapi
+ldap
+libgcrypt
+docbook
+libiconv
+pam
+```
+
+Please see `meson_options.txt` for full details of all Netatalk-specific options,
+and https://mesonbuild.com/Builtin-options.html for details of generic Meson
+options.
+
+## External software dependencies
+
+Required:
+
+  - libevent 2
+  - libgcrypt version 1.2.3 or later (for DHX2 UAM support)
+  - Berkeley DB version 4.6.0 or later
+  - perl
+
+Required for Spotlight support:
+  - talloc
+  - tracker version 0.12 or later
+  - bison
+  - flex
+
+Optional:
+
+  - Kerberos V              (for krbV UAM support)
+  - libressl or OpenSSL@1.1 (for Randnum and DHX UAM support)
+  - xsltproc                (for manpages and mananual documentation)
+  - Docbook XSL stylesheets (for manpages & manual documentation)
+  - GLib 2 and dbus-glib    (for afpstats support)
+  - cracklib                (for cracklib support)
+  - avahi or mDNSresponder  (for Zeroconf support)
+  - tcpwrap                 (for TCP wrapper support)
+
```

Comment:
Maybe tell the reader that more details can be found in the html manual at https://netatalk.io/stable/htmldocs/installation

Reply:
Fixed